README.md

```diff
@@ -231,7 +231,7 @@ You may also choose to pull from regional Amazon ECR repositories such as the fo
 
 Example regional image URI:
 ```
-328549459982.dkr.ecr.us-west-2.amazonaws.com/bottlerocket-update-operator:v0.2.1
+328549459982.dkr.ecr.us-west-2.amazonaws.com/bottlerocket-update-operator:v0.2.2
```

Comment:
Since it's just an example, I think we don't need update it next time when we release new version : )